### PR TITLE
add design document

### PR DESCRIPTION
**Pull request by ramsayleung**
as title

### CODE REVIEW

File: docs/design.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## 1. Basic Features 
+ProductType service provides following functions:
++ create productType
++ delete productType
++ update productType
++ get individual productType
++ gett all productType
+
+## 2. Model Design
+For ProducTType, there are such requirements:
```
Comment (davis999) [inline docs/design.md]: should be "ProductType".

File: docs/design.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+* multi-language
+label, inputTip in attributes of productType should be multiple language
+
```
Comment (davis999) [inline docs/design.md]: add attribute definition design.

File: docs/design.md (hunk 1 above)
Comment (ramsayleung) [reply docs/design.md]: fix typo

File: docs/design.md (hunk 2 above)
Comment (ramsayleung) [reply docs/design.md]: 1. add attribute definition design.

**Review by davis999 — APPROVED** [review]

**Pull request merged**